pkg/common/config/config.go

```diff
@@ -364,6 +422,17 @@ func validateConfig(cfg *Config) error {
 			vcConfig.Thumbprint = cfg.Global.Thumbprint
 		}
 
+		if vcConfig.IPFamily == "" {
+			vcConfig.IPFamily = cfg.Global.IPFamily
+		}
+
+		ipFamilyPriority, err := validateIPFamily(vcConfig.IPFamily)
+		if err != nil {
+			klog.Errorf("Invalid vcConfig IPFamily: %s", vcConfig.IPFamily)
```

Comment:
Log the error? Though in general do we need to log error if we eventually log the returned error?

Reply:
We can do both. Doesn't really matter.